File: ParkingPlace/app/src/main/java/com/rmj/parking_place/dto/ParkingPlaceChangesDTO.java
package com.rmj.parking_place.dto;

import java.util.List;

public class ParkingPlaceChangesDTO {
    private Long zoneId;
    private Long version;
    private List<ParkingPlaceDTO> parkingPlaceChanges;


    public ParkingPlaceChangesDTO() {

    }


    public Long getZoneId() {
        return zoneId;
    }

    public void setZoneId(Long zoneId) {
        this.zoneId = zoneId;
    }

    public Long getVersion() {
        return version;
    }

    public void setVersion(Long version) {
        this.version = version;
    }

    public List<ParkingPlaceDTO> getParkingPlaceChanges() {
        return parkingPlaceChanges;
    }

    public void setParkingPlaceChanges(List<ParkingPlaceDTO> parkingPlaceChanges) {
        this.parkingPlaceChanges = parkingPlaceChanges;
    }
}

File: ParkingPlaceServer/ParkingPlaceServer/DTO/ParkingPlaceChangesDTO.cs
using Newtonsoft.Json;
using ParkingPlaceServer.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ParkingPlaceServer.DTO
{
	public class ParkingPlaceChangesDTO
	{

		[JsonProperty("zoneId")]
		public long ZoneId { get; set; }


		[JsonProperty("version")]
		public long Version { get; set; }


		[JsonProperty("parkingPlaceChanges")]
		public List<ParkingPlaceDTO> ParkingPlaceChanges { get; set; }


		public ParkingPlaceChangesDTO()
		{

		}

		public ParkingPlaceChangesDTO (long zoneId, long version, List<ParkingPlaceDTO> parkingPlaceChanges)
		{
			ZoneId = zoneId;
			Version = version;
			ParkingPlaceChanges = parkingPlaceChanges;
		}
	}
}